### ISSUE TITLE
Why bbox (south and north) seems to be reverted in this record?

### ISSUE THREAD

**Issue by joanma747** [EOEPCA/open-science-catalog-metadata#72]
https://resource-catalogue.demo.eoepca.org/collections/metadata:main/items/S2A_MSIL1C_20210505T105031_N0300_R051_T31UES_20210505T144122.SAFE?f=xml

<gmd:southBoundLatitude>
<gco:Decimal>51.45118220579359</gco:Decimal>
</gmd:southBoundLatitude>
<gmd:northBoundLatitude>
<gco:Decimal>50.453523278919015</gco:Decimal>
</gmd:northBoundLatitude>

Is this a mistake in the data or in the application? 

Sorry if this is not the forum to report this.

**Comment by kalxas** [comment]
Yes, this repository is about ESA Open Science Catalogue, not for the demo EOEPCA instance, but I will investigate the issue and report back

**Comment by kalxas** [comment]
We need to check this part of the registration process:
https://github.com/EOEPCA/rm-data-access/blob/master/core/registrar_pycsw/metadata.py#L351-L356

**Comment by kalxas** [comment]
Looks like a problem with the product: https://finder.creodias.eu/files/Sentinel-2/MSI/L2A/2020/09/30/S2A_MSIL2A_20200930T092031_N0214_R093_T34SEF_20200930T122328.SAFE/MTD_MSIL2A.xml

**Comment by kalxas** [comment]
```
<Global_Footprint>
<EXT_POS_LIST>
37.046583050483974 20.99977508317119 37.04016090428803 22.234438031342126 36.05052821595199 22.218800020089912 36.05672396511734 20.999777933310174 37.046583050483974 20.99977508317119
</EXT_POS_LIST>
</Global_Footprint>
```

**Comment by kalxas** [comment]
Fix committed:

https://github.com/EOEPCA/rm-data-access/commit/30bf8e3abff069bd6fa5a84a05f506f8e1c46e28

**Comment by joanma747** [comment]
Thanks

**Comment by kalxas** [comment]
Fixed here now: https://resource-catalogue.develop.eoepca.org/collections/S2MSI1C/items/S2B_MSIL1C_20190910T095029_N0208_R079_T33UWQ_20190910T120910.SAFE?f=xml
